package.json

```diff
@@ -7,7 +7,15 @@
     "type": "git",
     "url": "git://github.com/vend/vend-number.git"
   },
-  "author": "Sarah Hui <sarah@vendhq.com>",
+  "contributors": [{
+    "name": "Sarah Hui",
+    "email": "sarah@vendhq.com",
+    "url": "https://github.com/sehsarah"
+  }, {
+    "name": "James Bunt",
+    "email": "james@vendhq.com",
+    "url": "https://github.com/unkillbob"
+  }],
```

Comment:
Sure I wasn't sure who's worked on it before, I only had the history of the copy Sarah made in weggie back in the day :+1: